paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/GenericTextFieldConfig.kt

```diff
@@ -0,0 +1,29 @@
+package com.stripe.android.paymentsheet.elements
+
+import androidx.compose.ui.text.input.KeyboardCapitalization
+import androidx.compose.ui.text.input.KeyboardType
+import com.stripe.android.paymentsheet.ElementType
+
+internal class GenericTextFieldConfig(
+    override val debugLabel: String = "generic_text",
+    override val label: Int,
+    override val keyboard: KeyboardType = KeyboardType.Text,
+    override val elementType: ElementType = ElementType.GenericText,
+    override val capitalization: KeyboardCapitalization = KeyboardCapitalization.None
+) : TextFieldConfig {
```

Comment:
Do all these properties need to be part of the constructor?

Also consider making `label` the first parameter in the constructor because it's the only property that doesn't have a default value.

Reply:
I compared this to the other configs in the code base and they are not constructor parameters there, so I will change it here.